app/models/concerns/mentionable.rb

```diff
@@ -0,0 +1,42 @@
+module Mentionable
```

Comment:
This is a well written concern. Simple to read, comprehend and does it's job cleanly.